src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -1,96 +1,17 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains MSBuild support for Razor.</Description>
-    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
 
-    <!-- This project doesn't have any code, so don't include it in the .nupkg -->
+    <!-- This project doesn't have any code, so don't include a binary in the .nupkg -->
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <EnableDefaultItems>false</EnableDefaultItems>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <EnableApiCheck>false</EnableApiCheck>
 
-    <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
-  </PropertyGroup>
-
-  <!--
-    Building this package is somewhat complicated because we need to Build or Publish some other projects
-    that have different TFM's including one with multiple TFMs.
-
-    We then need to include the output of those projects in our output directory (where it will be used
-    by tests) and in the nukpg.
-  -->
-  <!-- These are tools that need to be included in the package. -->
-
-  <PropertyGroup>
-    <ToolProject>..\Microsoft.AspNetCore.Razor.Tools\Microsoft.AspNetCore.Razor.Tools.csproj</ToolProject>
-    <TaskProject>Microsoft.AspNetCore.Razor.Tasks</TaskProject>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\$(TaskProject)\$(TaskProject).csproj" ReferenceOutputAssembly="false" />
-    <ProjectReference Include="$(ToolProject)" ReferenceOutputAssembly="false" Condition="'$(TargetFramework)' == 'netcoreapp2.0'" />
+    <Content Include="_._" Pack="true" />
```

Comment:
This package is entirely empty now. Adding some place holder items

Reply:
Would we still want the ability to allow users to override the SDK and get a newer version of the compiler? Or would that be handled by referencing the package directly.

Reply:
You would reference the Sdk package to get the new compiler \ code generation. I'll make sure it works E2E with a package reference